core/templates/app.constants.ts

```diff
@@ -16,6 +16,11 @@
  * @fileoverview Shared constants for the Oppia module.
  */
 
+export interface WARNING_TYPES_CONSTANT {
```

Comment:
I think this isn't the right place for declaring type interfaces. Let us create a new file `app-type.constants.ts` and add the type interfaces there.

Reply:
Done!